ocs_ci/ocs/resources/pod.py

```diff
@@ -1791,14 +1806,19 @@ def delete_osd_removal_job(osd_id):
         bool: True, if the ocs-osd-removal job deleted successfully. False, otherwise
 
     """
-    osd_removal_job = get_job_obj(
-        f"ocs-osd-removal-{osd_id}", namespace=defaults.ROOK_CLUSTER_NAMESPACE
-    )
+    ocp_version = float(get_ocp_version())
+    ocs_version = float(config.ENV_DATA["ocs_version"])
+    if ocp_version >= 4.6 and ocs_version >= 4.7:
```

Comment:
We should be using semantic_version according to Petr's email a couple weeks back. This will basically prevent comparisons like this from breaking once ocs_version reaches something like 4.10 or 4.11. This code expects an ocs_version of 4.10 to be greater than 4.7 but the float 4.10 is less than 4.7. By using semantic_version, we can avoid this.

@clacroix12 ,Cody comment:)

Reply:
Fixed